docs/load-balancer-annotations.md

```diff
@@ -49,8 +49,8 @@ spec:
 | `service.beta.kubernetes.io/oci-load-balancer-tls-secret`                       | The TLS secret to install on the load balancer listeners which have SSL enabled.                                                                                                                                                                                                 | `N/A`                                            |
 | `oci.oraclecloud.com/oci-network-security-groups`                               | Specifies Network Security Groups' OCIDs to be associated with the loadbalancer. Please refer [here][8] for NSG details.                                                                                                                                                         | `N/A`                                            |
 | `oci.oraclecloud.com/loadbalancer-policy`                                       | Specifies loadbalancer traffic policy for the loadbalancer. To get a list of valid policies, use the [`ListPolicies`][7] operation.                                                                                                                                              | `"ROUND_ROBIN"`                                  |
-| `oci.oraclecloud.com/initial-defined-tags-override`                             | Specifies one or more Defined tags to apply to the OCI Load Balancer.                                                                                                                                                                                                            | `N/A`                                            |
-| `oci.oraclecloud.com/initial-freeform-tags-override`                            | Specifies one or more Freeform tags to apply to the OCI Load Balancer.                                                                                                                                                                                                           | `N/A`                                            |
+| `oci.oraclecloud.com/initial-defined-tags-override`                             | Specifies one or more Defined tags to apply to the OCI Load Balancer. Valid values: `'{"tag1": "value1", "tag2": "value2"}'`                                                                                                                                                     | `N/A`                                            |
```

Comment:
Defined tags require another level
'{"namespace1": {"tag1": "value1", "tag2": "value2"}}'

Reply:
OK, I did that